include/bio_ik/frame.h

```diff
@@ -51,12 +51,13 @@ typedef tf2::Vector3 Vector3;
 struct alignas(32) Frame
 {
     Vector3 pos;
-    double __padding[4 - (sizeof(Vector3) / sizeof(double))];
+    // TODO(wyattrees): is __padding needed? rename without double underscore?
```

Comment:
This seems like it might be for getting the Quaternion to be on some address multiple so it is optimized for simd instructions. Honestly I feel like much of these optimizations here are not justified without benchmarks and make the code considerably harder to read without any justification for their existance.